editor/css/main.css

```diff
@@ -345,6 +345,7 @@ select {
 		float: left;
 		cursor: pointer;
 		padding-right: 8px;
+		background: inherit;
```

Comment:
I'm not really comfortable with these background related changes. Do you mind focusing on just the scrollbar issue?

Reply:
`inherit` simply means 'just use the same value of my parent element'.

Its upside is that, in our case ... *the background property is now dominant by single element, the `#menubar`* ... good for maintenance. 

Without `inherit`, the background property will be copied and pasted many places, in our case, this background property **has to** set on `#menubar`, `#menubar title`, `#menubar options option` etc, note that [we have to repeat that for dark mode too](https://github.com/mrdoob/three.js/blob/79c34909e6af9250c38182665e3bd4896149f0ea/editor/css/main.css#L574) (effort x2)